code/scripting/api/objs/subsystem.cpp

```diff
@@ -607,8 +613,6 @@ ADE_FUNC(fireWeapon, l_Subsystem, "[number TurretWeaponIndex = 1, number FlakRan
 
 	bool rtn = turret_fire_weapon(wnum, sso->ss, OBJ_INDEX(sso->objp), &gpos, &gvec, NULL, flak_range);
 
-	sso->ss->turret_next_fire_pos++;
```

Comment:
What purpose does removing this line serve?

Reply:
I thought it was redundant, but upon closer inspection it actually isn't.  I'll put it back.